### PR TITLE
feat(pci.ai.apps) : ai apps pages

### PR DESCRIPTION
**Pull request by XciD**
| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | yes
| Tickets          |  MANAGER-8135
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [x] Breaking change is mentioned in relevant commits

## Description

AI Apps for public cloud in Alpha

### :flags: Translations

- [x] TRANS-39452
- [x] TRANS-40159

### CODE REVIEW

File: packages/manager/modules/pci/src/projects/project/ai/tokens/tokens.routing.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+        }),
+
+      updateToken: /* @ngInject */ ($state, projectId) => (tokenId) =>
+        $state.go('pci.projects.project.ai.tokens.update', {
+          projectId,
+          tokenId,
+        }),
+
+      goToTokens: ($state, CucCloudMessage, projectId) => (
+        message = false,
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/ai/tokens/tokens.routing.js:35]: Could the Sonarcloud feedback be taken in account here please?

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-attach/app-attach.html [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+<!--App storage labels-->
+<div class="mb-4">
+    <div class="oui-box oui-box_light pl-0 pb-2 border-0">
+        <!--Manage labels-->
+        <div class="oui-inline-adder mb-2">
+            <div
+                class="oui-inline-adder__form"
+                data-ng-repeat="volume in $ctrl.volumes track by $index"
+            >
+                <fieldset class="oui-inline-adder__fieldset">
```
Comment (JeremyDec) [inline packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-attach/app-attach.html:31]: Same here please, maybe add `legend` for the fieldsets

Comment (XciD) [reply packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-attach/app-attach.html:31]: I've check all the `<fieldset class="oui-inline-adder__fieldset">` none had a legend.

Comment (JeremyDec) [reply packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-attach/app-attach.html:31]: Yes, it is not done for the moment, but it should ;)

Comment (XciD) [reply packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-attach/app-attach.html:31]: I understand, but

- I don't know what to put
- I don't know the look of it after.

So, as I'm the first, I will not do it without an example

File: packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-resources/translations/Messages_fr_FR.json [hunk 3]
```diff
@@ -0,0 +1,25 @@
+{
+  "pci_app_add_resources_usecase": "Type de d'instances",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ai/apps/add/components/app-resources/translations/Messages_fr_FR.json:2]:
```suggestion
  "pci_app_add_resources_usecase": "Type d'instances",
```

File: packages/manager/modules/pci/src/projects/project/ai/apps/components/delete-app/translations/Messages_fr_FR.json [hunk 4]
```diff
@@ -0,0 +1,8 @@
+{
+  "pci_app_delete_app_title": "Suppression du app",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ai/apps/components/delete-app/translations/Messages_fr_FR.json:2]: **question:**
```suggestion
  "pci_app_delete_app_title": "Suppression de l'app",
```

File: packages/manager/modules/pci/src/projects/project/ai/apps/components/delete-app/translations/Messages_fr_FR.json [hunk 5]
```diff
@@ -0,0 +1,8 @@
...
+  "pci_app_delete_app_submit_label": "Confirmer",
+  "pci_app_delete_app_cancel_label": "Annuler",
+  "pci_app_delete_app_content": "Êtes-vous sûr·e de vouloir supprimer l'app {{ appName }} ?",
+  "pci_app_delete_app_success": "Votre app a bien été supprimé.",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ai/apps/components/delete-app/translations/Messages_fr_FR.json:6]:
```suggestion
  "pci_app_delete_app_success": "Votre app a bien été supprimée.",
```

File: packages/manager/modules/pci/src/projects/project/ai/apps/onboarding/translations/Messages_fr_FR.json [hunk 6]
```diff
@@ -0,0 +1,17 @@
+{
+  "pci_app_title": "AI Apps",
+  "pci_app_onboarding_content1": "Vos applications et modèles AI déployés en haute disponibilité.",
+  "pci_app_onboarding_content2": "Choisissez parmi un catalogue ou déployez vos propres images Docker.",
+  "pci_app_onboarding_content3": "Vous êtes data scientist ou développeur et souhaitez déployer une application ou un modèle d'intelligence artificielle en quelques minutes ? Sélectionnez parmi un catalogue d'AI Apps ou importez votre propre application containerisée. OVHcloud s'occupe de son déploiement en haute disponibilité et de sa sécurisation. Disposez de ressources CPU ou GPU, d'outils de suivi en temps réel, et d'une tarification simple, facturée à l'usage. Libre à vous ensuite de vous y connecter par API ou via une interface Web HTTP.",
+  "pci_app_onboarding_action_label": "Déployer une AI App",
+  "pci_app_guides_documentation_title": "Documentation complète AI Apps",
+  "pci_app_guides_documentation_description": "Retrouvez toute la documentation d'AI Apps.",
+  "pci_app_guides_definition_title": "Définition AI Apps",
+  "pci_app_guides_definition_description": "Découvrez ce qu'est un AI App chez OVHcloud.",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ai/apps/onboarding/translations/Messages_fr_FR.json:10]:
```suggestion
  "pci_app_guides_definition_description": "Découvrez ce qu'est une AI App chez OVHcloud.",
```

File: packages/manager/modules/pci/src/projects/project/ai/apps/onboarding/translations/Messages_fr_FR.json [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+  "pci_app_guides_start_title": "Démarrer avec AI Apps",
+  "pci_app_guides_start_description": "Apprenez comment démarrer votre App rapidement dans votre espace client.",
+  "pci_app_guides_share_title": "Partager une AI App",
+  "pci_app_guides_share_description": "Apprenez comment partager un AI App avec d'autres développeurs.",
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ai/apps/onboarding/translations/Messages_fr_FR.json:14]:
```suggestion
  "pci_app_guides_share_description": "Apprenez comment partager une AI App avec d'autres développeurs.",
```

File: packages/manager/modules/pci/src/projects/project/ai/apps/onboarding/translations/Messages_fr_FR.json [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+  "pci_app_guides_object_storage_access_title": "Accéder à vos datas depuis votre object storage",
+  "pci_app_guides_object_storage_access_description": "Découvrez comment accéder à vos datas de App avec votre object storage."
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ai/apps/onboarding/translations/Messages_fr_FR.json:16]: Sound correct?

File: packages/manager/modules/pci/src/projects/project/ai/tokens/update/translations/Messages_en_GB.json [hunk 9]
```diff
@@ -0,0 +1,8 @@
+{
+  "pci_ai_tokens_update_success": "Votre token a été regénéré avec succès, enregistrez-le",
+  "pci_ai_tokens_update_error": "Une erreur est survenue durant la regénération du token : {{ message }}",
+  "pci_ai_tokens_update": "Regénération du token",
+  "pci_ai_tokens_update_common_update": "Regénérer",
+  "pci_ai_tokens_update_common_cancel": "Annuler",
+  "pci_ai_tokens_update_warning": "Êtes vous sûr de vouloir regénérer {{ tokenId }} ? Votre ancien token ne sera plus valide."
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ai/tokens/update/translations/Messages_en_GB.json:7]:
```suggestion
  "pci_ai_tokens_update_warning": "Êtes-vous sûr(e) de vouloir regénérer {{ tokenId }} ? Votre ancien token ne sera plus valide."
```

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -11,7 +11,7 @@ pay attention to a few things:
 
 # Coding and documentation Style
 
-- Code must be formated with `yarn run format`
+- Code must be formatted with `yarn run format`
```
Comment (antleblanc) [inline CONTRIBUTING.md:14]: **nitpick:** Changes should be extracted into its on commit `docs: remove typo in CONTRIBUTING.md file` (as an example)

File: packages/manager/modules/pci/src/projects/project/ai/tokens/update/translations/Messages_en_GB.json [hunk 11]
```diff
@@ -0,0 +1,8 @@
+{
```
Comment (antleblanc) [inline packages/manager/modules/pci/src/projects/project/ai/tokens/update/translations/Messages_en_GB.json:1]: :warning: **issue(blocking):** File must be named as `Messages_fr_FR.json`

**Review by antleblanc — CHANGES_REQUESTED** [review]

**Review by antleblanc — DISMISSED** [review]

**Review by JeremyDec — DISMISSED** [review]

**Comment by antleblanc** [comment]
Retrieving translations.

**Review by antleblanc — DISMISSED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**